{
  "modified": "2025-05-22T12:33:43Z",
  "published": "2025-05-22T12:33:43Z",
  "schema_version": "1.5.0",
  "id": "MAL-2025-4233",
  "summary": "Malicious code in pystile (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pystile",
        "purl": "pkg:pypi/pystile"
      },
      "versions": [
        "0.0.4",
        "0.0.5",
        "0.0.6",
        "0.0.7",
        "0.1.1"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2025-02590",
        "import_time": "2025-05-22T14:06:37.054072341Z",
        "modified_time": "2025-05-22T12:33:43Z",
        "sha256": "b953273ca310ebd0d2eb0442f54651302d72d21f7469f8c1b349786d148e1417",
        "source": "reversing-labs",
        "versions": [
          "0.0.4",
          "0.0.5",
          "0.0.6",
          "0.0.7",
          "0.1.1"
        ]
      }
    ]
  }
}
